mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -1228,6 +1416,357 @@ public struct EpollEvent {
 		[FieldOffset (4)]
 		public ulong u64;
 	}
+
+	[Map]
+	[StructLayout (LayoutKind.Explicit, Size=4)]
+	public struct InAddr {
+		[FieldOffset (0)] uint addr;
```

Comment:
Why do we need the `[FieldOffset]` here? There's only one field!

Related: Why do we need to explicitly set the size to 4?